server/src/main/java/io/openbytes/odi/interfaces/controller/api/DatasetController.java

```diff
@@ -1,27 +1,31 @@
 package io.openbytes.odi.interfaces.controller.api;
 
 import io.openbytes.odi.application.DatasetService;
+import io.openbytes.odi.application.DatasetFileService;
+import io.openbytes.odi.domain.DatasetFile;
 import io.openbytes.odi.interfaces.Result;
 import io.openbytes.odi.interfaces.vo.DatasetVO;
 import io.swagger.annotations.Api;
 import io.swagger.annotations.ApiOperation;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import javax.annotation.Resource;
+import java.util.List;
 import java.util.Optional;
 
-@Api(tags = "【web】Dataset")
+@Api(tags = "[web] Dataset")
 @RequestMapping("/api/v1/datasets")
 @RestController
 public class DatasetController {
 
     @Resource
     private DatasetService service;
 
+    @Resource
+    private DatasetFileService odiResourceService;
```

Comment:
odiResourceService should be renamed to fileService

Reply:
Done